flags/ios_nocodesign.cmake

```diff
@@ -7,46 +7,54 @@ else()
   set(POLLY_FLAGS_IOS_NOCODESIGN_CMAKE_ 1)
 endif()
 
-# Verify XCODE_XCCONFIG_FILE
-set(
-    _polly_xcode_xcconfig_file_path
-    "${CMAKE_CURRENT_LIST_DIR}/../scripts/NoCodeSign.xcconfig"
-)
-
-get_filename_component(
-    _polly_xcode_xcconfig_file_path
-    "${_polly_xcode_xcconfig_file_path}"
-    ABSOLUTE
-)
-
-get_filename_component(
-    _polly_xcode_xcconfig_file_path_env
-    "$ENV{XCODE_XCCONFIG_FILE}"
-  ABSOLUTE
-)
-
-if(NOT EXISTS "${_polly_xcode_xcconfig_file_path_env}")
-  polly_fatal_error(
-      "Path specified by XCODE_XCCONFIG_FILE environment variable not found"
-      "($ENV{XCODE_XCCONFIG_FILE})"
-      "Use this command to set: "
-      "    export XCODE_XCCONFIG_FILE=${_polly_xcode_xcconfig_file_path}"
-  )
+if("${IOS_SDK_VERSION}" VERSION_GREATER_EQUAL "13.0")
```

Comment:
Do you also need to check the CMake version here?

Reply:
No, because [`CMAKE_XCODE_ATTRIBUTE_<an-attribute>`](https://cmake.org/cmake/help/v3.1/variable/CMAKE_XCODE_ATTRIBUTE_an-attribute.html) exists since cmake 3.1.

My change is only set an additional XCode property `CODE_SIGN_IDENTITY ""` to create a binary/library without signing it. It is neede since iOS sdK 13. Before it was enough to set only `CODE_SIGNING_REQUIRED NO`.

Btw. This works also for "older" xcode. My assumption is, we didn't need this `XCODE_XCCONFIG_FILE` approach ... but I'm not sure why this approach is introduced. It is enough to set the XCode propery via `CMAKE_XCODE_ATTRIBUTE_<an-attribute>` -->> `set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED NO)`